mache/spack/pm-cpu_gnu_mpich.yaml

```diff
@@ -32,21 +32,46 @@ spack:
       - spec: curl@7.66.0
         prefix: /usr
       buildable: false
+    cmake:
+      externals:
+      - spec: cmake@3.24.3
+        prefix: /global/common/software/nersc/pm-2022q4/spack/linux-sles15-zen/cmake-3.24.3-k5msymx/
+      buildable: false
     gettext:
       externals:
       - spec: gettext@0.20.2
         prefix: /usr
       buildable: false
+    gmake:
+      externals:
+      - spec: gmake@4.2.1
+        prefix: /usr
+      buildable: false
+    libuv:
+      externals:
+      - spec: libuv@1.44.2
+        prefix: /usr
+      buildable: false
     libxml2:
       externals:
       - spec: libxml2@2.9.7
         prefix: /usr
       buildable: false
+    m4:
+      externals:
+      - spec: m4@6.1.20180317
```

Comment:
Here and elsewhere, the verison is not correct (copied from ncurses) and should be fixed.